packages/react-generator/src/templates/react-template/template-files/src/http/axios.js

```
@@ -0,0 +1,139 @@
+
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+
+export default (config) => {
+  const instance = axios.create(config)
+  const defaults = {}
+  let mock
+
+  if (typeof MockAdapter.prototype.proxy === 'undefined') {
+    MockAdapter.prototype.proxy = function ({ url, config = {}, proxy, response, handleData } = {}) {
+      let stream = this
```

ESLint

[error] Unexpected aliasing of 'this' to local variable.

(@typescript-eslint/no-this-alias)